src/application/tactics/line_detected.c

```diff
@@ -40,7 +40,7 @@ static void back_up_timer_expired(tiny_timer_group_t* group, void* context) {
   motor_power_t left_power;
   motor_power_t right_power;
 
-  if(the_left_line_was_detected(self)) {
+  if(self->left_line_was_detected) {
```

Comment:
This makes it capture the line at the point when it starts the maneuver, right?

Reply:
Yeah, previously it would look at whether the line was detected _after_ it backed up (spoiler: it better not be)